homeassistant/components/cover/fibaro.py

```diff
@@ -0,0 +1,91 @@
+"""
+Support for Fibaro cover - curtains, rollershutters etc.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/cover.fibaro/
+"""
+import logging
+
+from homeassistant.components.cover import CoverDevice, ENTITY_ID_FORMAT, \
+    ATTR_POSITION, ATTR_TILT_POSITION
+from homeassistant.components.fibaro import (
+    FIBARO_CONTROLLER, FIBARO_DEVICES, FibaroDevice)
+
+DEPENDENCIES = ['fibaro']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Fibaro covers."""
+    if discovery_info is None:
+        return
+
+    add_entities(
+        [FibaroCover(device, hass.data[FIBARO_CONTROLLER]) for
+         device in hass.data[FIBARO_DEVICES]['cover']], True)
+
+
+class FibaroCover(FibaroDevice, CoverDevice):
+    """Representation a Fibaro Cover."""
+
+    def __init__(self, fibaro_device, controller):
+        """Initialize the Vera device."""
+        FibaroDevice.__init__(self, fibaro_device, controller)
```

Comment:
You're inheriting from `FibaroDevice`, so you can just do:

```
super().__init__(...)
```

Reply:
Ok, thanks